containers/ecr-viewer/tests/unit/app/view-data/services/labsService.test.tsx

```diff
@@ -289,188 +290,392 @@ describe("LabsService tests", () => {
 
-        expect(result).toEqual("09/28/2000 4:59\u00A0PM\u00A0EDT");
-      });
+      const result = searchResultRecord(labHTMLJson, searchKey);
 
-      it("extracts returns noData if unavailable", () => {
-        const fieldName = "Analysis Time";
+      expect(result).toStrictEqual(expectedResult);
+    });
 
-        const result = returnAnalysisTime({}, fieldName);
+    it("returns an empty string of results if none are found for search key", () => {
+      const invalidSearchKey = "foobar";
+      const expectedResult = "";
 
-        expect(result).toEqual(noData);
-      });
+      const result = searchResultRecord(labHTMLJson, invalidSearchKey);
 
-      it("Concats date times if multiple passed", () => {
-        const fieldName = "Analysis Time";
+      expect(result).toEqual(expectedResult);
+    });
+  });
 
-        const result = returnAnalysisTime(
-          {
-            resultId: "Result.1.2.3.4.5",
-            resultName: "Stool Pathogens, NAAT, 12 to 25 Targets",
-            tables: [
-              [
-                {
-                  "Analysis Time": {
-                    value: (
-                      <p>
-                        <span>
-                          <i>09/28/2000 1:59:00 PM PDT</i>
-                        </span>
-                        <span>
-                          <i>09/28/2000 2:59:00 PM PDT</i>
-                        </span>
-                      </p>
-                    ),
-                    metadata: {},
-                  },
+  describe("returnAnalysisTime", () => {
+    it("extracts and formats correct field value from within a lab report", () => {
+      const fieldName = "Analysis Time";
+
+      const result = returnAnalysisTime(labReportNormalJsonObject, fieldName);
+
+      expect(result).toEqual("09/28/2000 4:59\u00A0PM\u00A0EDT");
+    });
+
+    it("extracts returns noData if unavailable", () => {
+      const fieldName = "Analysis Time";
+
+      const result = returnAnalysisTime({}, fieldName);
+
+      expect(result).toEqual(noData);
+    });
+
+    it("Concats date times if multiple passed", () => {
+      const fieldName = "Analysis Time";
+
+      const result = returnAnalysisTime(
+        {
+          resultId: "Result.1.2.3.4.5",
+          resultName: "Stool Pathogens, NAAT, 12 to 25 Targets",
+          tables: [
+            [
+              {
+                "Analysis Time": {
+                  value: (
+                    <p>
+                      <span>
+                        <i>09/28/2000 1:59:00 PM PDT</i>
+                      </span>
+                      <span>
+                        <i>09/28/2000 2:59:00 PM PDT</i>
+                      </span>
+                    </p>
+                  ),
+                  metadata: {},
                 },
-              ],
+              },
             ],
-          },
-          fieldName,
-        );
+          ],
+        },
+        fieldName,
+      );
```

Comment:
the diff got super wonky here, but things didn't actually change outside of the added tests and simplifying the describe nests within the lab service tests